index.js

```diff
@@ -112,7 +112,9 @@ module.exports = (options = {}) => {
 		})();
 	}, function (callback) {
 		this.size = totalSize.values().next().value;
-		this.prettySize = prettyBytes(this.size);
+		if (this.size) {
+			this.prettySize = prettyBytes(this.size);
+		}
```

Comment:
`prettySize` is documented to always exist, so it should fall back to `0`. Just do a ternary.